webapp/app.py

```diff
@@ -46,6 +46,12 @@ def pluralize(count):
             return 's'
         return ''
 
+    @app.context_processor
+    def inject_external_urls():
+        return dict(external_urls={
+            'api': 'https://api.jujucharms.com/charmstore/v5/'
```

Comment:
This has a trailing slash, so every link in the template has double slashes.

Reply:
Ah yes, well spotted!